helm/cluster-gcp/templates/_bastion.tpl

```diff
@@ -43,34 +72,21 @@ metadata:
   labels:
     cluster.x-k8s.io/role: bastion
     {{- include "labels.common" $ | nindent 4 }}
-  name: {{ include "resource.default.name" $ }}-bastion-{{ include "hash" (dict "data" .Values.bastion "global" .) }}
+  name: {{ include "resource.default.name" $ }}-bastion-{{ include "hash" (dict "data" (include "bastion-machinetemplate-spec" $) "global" .) }}
```

Comment:
but does this actually change when values change as well? is the hash computed after all values are filled?

Reply:
Yes. It's based on the rendered template

Reply:
ack